.readthedocs.yaml

```
@@ -12,7 +12,7 @@ build:
   commands:
     - python -m pip install --upgrade --no-cache-dir pip setuptools
     - python -m pip install --upgrade -r ./requirements.in -c ./requirements.txt
-    - nikola build
+    - nikola build -- --conf=dev.conf.py
```

Shouldn't this be
```suggestion
    - nikola build --conf=dev.conf.py
```
?